__tests__/boot.ts

```diff
@@ -361,6 +363,32 @@ describe('logged in boot', () => {
       shouldLogout: true,
     });
   });
+
+  it('should not dispatch whoami when jwt is available', async () => {
+    const accessToken = await signJwt(
+      {
+        userId: '1',
+        roles: [],
+      },
+      15 * 60 * 1000,
+    );
```

Comment:
Should there be a test if the jwt has expired?

Reply:
There's a test with no jwt, do you think we need an expired one as well?

Reply:
Probably a good case actually yes :)

Reply:
Doesn't hurt to have one 🙃

Reply:
I'll create one quickly

Reply:
Done! You can check again

Reply:
![blueblob_jump](https://github.com/dailydotdev/daily-api/assets/1681525/66e10a35-3ccc-497e-b306-2908bd3bf937)